tests/unit/test_hsm.cpp

```diff
@@ -79,7 +87,36 @@ void HSMTest::SetUp()
     // TODO: Get rid of the uninteresting calls by default here somehow...
 }
 
-void HSMTest::TearDown() { openssl::OpenSSLLibMockManager::destroy(); }
+void HSMTest::TearDown()
```

Comment:
Cannot you use this new helper also for testHSMEngine now?

Reply:
Yeah. That makes testHSMEngine test redundant so I deleted it. initialiseEngine and testHSMEngine both test only the constructor.